src/shared/components/Placeholder/Placeholder.tsx

```diff
@@ -22,6 +22,7 @@ const pulse = keyframes`
 `
 
 const Placeholder = styled.div<PlaceholderProps>`
+  display: 'inline-flex';
```

Comment:
`'inline-flex'` is an invalid property value here so it's not taken into account at all. If you wanted to use that, you'd need to remove the apostrophes from the value, but I don't think this property is actually needed here?